bitnami/mariadb-galera/values-production.yaml

```diff
@@ -158,11 +158,14 @@ ldap:
   ## Enable LDAP support
   ##
   enabled: false
-  # uri:
-  # base:
-  # binddn:
-  # bindpw:
-  # bslookup:
+  uri:
```

Comment:
Why are we un-commenting this if the values are empty?

Reply:
Because they're ignored as far as `ldap.enabled=false` and we want to avoid to use the `default` function in the templates.